setup.cfg

```diff
@@ -1,12 +1,59 @@
 [metadata]
+name = oauthlib
+version = attr: oauthlib.__version__
+author = The OAuthlib Community
+author_email = idan@gazit.me
+description = A generic, spec-compliant, thorough implementation of the OAuth request-signing logic
+license = BSD-3-Clause
 license_files = LICENSE
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+maintainer = Ib Lundgren
```

Comment:
```suggestion
maintainer = Asif Saif Uddin, Jonathan Huot
```

Reply:
@JonathanHuot Do you want your email address published as a Maintainer?